model/semconv/v1.6.1/generated_trace.go

```diff
@@ -14,7 +14,7 @@
 
 // Code generated from semantic convention specification. DO NOT EDIT.
 
-package semconv // import "go.opentelemetry.io/collector/model/semconv/v1.6.1"
```

Comment:
Should this be kept?

Reply:
Looks like the old version of the porto tool used to keep this line in generated files and now it's removed. I'll make sure this PR doesn't change it

Reply:
Updated